ros2_batch_job/__main__.py

```diff
@@ -132,9 +132,10 @@ def main(sysargv=None):
             ]
 
     # There are no debug packages available for PyQt5 and PySide, so we
-    # can't build/run rqt_graph on Windows debug builds.
+    # can't build/run rqt_graph or rqt_py_common on Windows debug builds.
```

Comment:
It seems that the package built just fine. It only seems to be failing the tests since there is no debug version of PySide2 available?

Reply:
While the package will build (it is pure python), there is no hope that it will ever run without a debug version of `PyQt5.QtCore` and `PySide2` (they are required for both the tests and runtime, via an import of `python_qt_binding`).

Reply:
Please update the comment then to state the actual reason why it is blacklisted.

Reply:
Done in 71c71ae